.github/pull_request_template.md

```diff
@@ -25,25 +25,60 @@
 
 - ...
 
-## 🚀 Has this modified a publishable library?
+## Pre-Merge Checklist 🚀
+
+### 🚢 Has this modified a publishable library?
 
 <!-- Confirm you have taken the necessary action to record a changeset or publish a change, as appropriate -->
-<!-- AT LEAST ONE box must be checked. Do not delete this section! see libs/README.md for info on publishing -->
+<!-- Remove the initial placeholder, and uncomment AT LEAST ONE of these. This MUST be done prior to merging. Do not delete this section! see libs/README.md for info on publishing -->
 
 This PR:
 
-- [ ] modifies an **npm**-publishable library and **I have added a changeset file(s)**
-- [ ] modifies a **Cargo**-publishable library and **I have amended the version**
-- [ ] modifies a **Cargo**-publishable library, but **it is not yet ready to publish**
-- [ ] modifies a **block** that will need publishing via GitHub action once merged
-- [ ] does not modify any publishable blocks or libraries, or modifications do not need publishing
-- [ ] I am unsure / need advice
+- [ ] **⚠️ This is a placeholder, please uncomment at least one of the lines following this one and then delete this. ⚠️**
```

Comment:
I opted for this pattern to try and solve your bugbear, @CiaranMn, about the number of "incomplete tasks" that appear for given PRs. This way we should be able to make sure that every checkbox has been checked

Reply:
omg yes, this was bothering me so much